docs/getting-started.stories.mdx

```diff
@@ -61,37 +61,41 @@ then the `GlobalStyle` component could be placed next to the content of the app:
 
 #### Fonts
 
-##### CDN
+Carbon requires the `Sage UI` and `CarbonIcons` fonts to display correctly. These will **_not_** be automatically loaded by Carbon, so you will required to load these into your application yourself.
 
-`carbon-react` requires the `Lato` and `CarbonIcons` fonts to display correctly.
+##### CDN
 
-To embed the `Lato` font, copy the code below into the `<head>` of your HTML.
+To embed the `Sage UI` font, copy the code below into the `<head>` of your HTML:
 
+```html
+<link
+  rel="preload stylesheet"
+  href="https://fonts.sage.com/sage.css"
+  as="font"
+  type="text/css"
+  crossorigin="anonymous"
+/>
 ```
-<link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900" rel="stylesheet">
-```
+
+The `CarbonIcons` font file is included within Carbon and just requires a `@font-face` to be defined for it (see under [Self Hosted](#self-hosted))
 
 ##### Self Hosted
 
-Please be aware that the Google Fonts CDN provides the optimal stylesheet based on the browser `user-agent` string to limit "flash of unstyled text". It's important that you fully understand the [technical considerations](https://developers.google.com/fonts/docs/technical_considerations) before hosting any font yourself.
+The font face definitions for `Sage UI` are already defined within the `@sage/design-tokens` package. Therefore once the package is installed the fonts can be imported into your code like so:
 
-The `Lato` font can be downloaded from [Google Fonts](https://fonts.google.com/specimen/Lato).
+###### CSS
 
 ```css
-@font-face {
-  font-family: "Lato";
-  src: url("./fonts/Lato-Regular.woff2") format("woff2");
-  src: url("./fonts/Lato-Regular.woff") format("woff");
-  src: url("./fonts/Lato-Regular.ttf") format("truetype");
-  src: url("./fonts/Lato-Regular.eot") format("embedded-opentype");
-  font-weight: normal;
-  font-style: normal;
-}
+@import "~@sage/design-tokens/assets/fonts/sageui.css";
 ```
 
-_Note: You'll need to define a `@font-face` for each font-weight._
+###### JS
+
+```js
+import fonts from "@sage/design-tokens/assets/fonts/sageui.css";
+```
 
-The path to the `CarbonIcons` font is `carbon-react/lib/style/fonts/carbon-icons-webfont.woff`.
+To load `CarbonIcons`, add the following css to your main stylesheet or in the `<head>` of your HTML wrapped in `<style>` tags:
```

Comment:
What do you think about creating an `icons.css` file, just like design tokens have. That way the process for installing the font is the same.

Reply:
I agree, this should keep things simple during setup 👍🏼